srcdoc/configure/server_timing_directives.mt

```diff
@@ -0,0 +1,121 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Server Timing Directives")->(sub {
+
+<p>
+The server timing handler turns on the <a href="https://www.w3.org/TR/server-timing/">Server Timing</a> feature.
+This document describes the configuration directives of the handler.
+</p>
+
+? $ctx->{directive_list}->()->(sub {
+
+<?
+my $access_log_ext_link = sub {
+    my $name = shift;
+    return $_mt->filter(sub {
+        "<a href=\"configure/access_log_directives.html#$name\"><code>$name</code></a>"
+    })->(sub {});
+};
+$ctx->{directive}->(
+    name    => "server-timing",
+    levels  => [ qw(global host path extension) ],
+    default => 'OFF',
+    since   => '2.3',
+    desc    => <<'EOT',
+A flag indicating how <code>server-timing</code> header and trailer should be served.
+EOT
+)->(sub {
+?>
+
+<p>
+If the argument is <code>ON</code>, H2O adds the <code>server-timing</code> header and trailer (if possible) to the response. The header is always added, but the trailer isn't if all of the following conditions are met.
+<ul>
+<li>The protocol used for the request is HTTP/1.1</li>
+<li><code>transfer encoding</code> of the response is not <code>chunked</code></li>
+</ul>
+</p>
+
+<p>
+If the value is <code>ENFORCE</code>, H2O forces the response to use chunked encoding by removing <code>content-length</code> header.
+</p>
+
+<p>
+If the value is <code>OFF</code>, the feature is disabled.
+</p>
+
+<p>
+The <code>server-timing</code> header and trailer include the following metrics. For now, all metrics have the <code>dur</code> attribute whose values are exactly equal to one of Access Log Extensions with similar names. To get the meaning of each <code>dur</code> attribute, see <a href="https://h2o.examp1e.net/configure/access_log_directives.html#access-log"><code>access-log</code></a>.
```

Comment:
Tweaks that might be worthwhile:

1) `s/include/includes/`
2) add "the" before `Access Log Extensions`